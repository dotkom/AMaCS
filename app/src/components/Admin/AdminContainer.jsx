import React from "react";

export default class AdminContainer extends React.Component {
  render() {
    return(
      <div>
        <p>"AdminContainer"</p>
      </div>
    )
  }
}