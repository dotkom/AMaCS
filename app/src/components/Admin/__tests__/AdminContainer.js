import React from 'react';
import { shallow } from 'enzyme';

import AdminContainer from '../AdminContainer';

describe('AdminContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <AdminContainer />
    )
    expect(wrapper).toMatchSnapshot();
  });
});
