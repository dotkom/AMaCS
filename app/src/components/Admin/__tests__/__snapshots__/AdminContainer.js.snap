// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminContainer renders correctly 1`] = `
<div>
  <p>
    "AdminContainer"
  </p>
</div>
`;
